tikv-configuration-file.md

```diff
@@ -1159,11 +1159,16 @@ import 相关的配置项。
 
 ### `wait-for-lock-timeout`
 
-+ 悲观事务在 TiKV 中等待其他事务释放锁的最长时间，单位为毫秒。若超时则会返回错误给 TiDB 并由 TiDB 重试加锁，语句最长等锁时间由 `innodb_lock_wait_timeout` 控制。
-+ 默认值：1000
```

Comment:
The config item is an integer, not string, why change it?

Reply:
It's a string in v4.0, and it's backward-compatible. https://github.com/tikv/tikv/pull/7197